packages/server/lib/hooks/hooks.ts

```diff
@@ -167,14 +158,13 @@ export const connectionRefreshFailed = async ({
     template: ProviderTemplate;
     config: IntegrationConfig;
     authError: { type: string; description: string };
-    activityLogId: number;
     logCtx: LogContext;
 }): Promise<void> => {
     await errorNotificationService.auth.create({
         type: 'auth',
         action: 'token_refresh',
         connection_id: connection.id!,
-        activity_log_id: activityLogId,
+        activity_log_id: -1,
```

Comment:
This will be removed soon, just want to avoid a db migration along with code change